src/ext/tls_internal.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * This is a mock of tls_internal.h filling out all the functions that are
+ * called by tls_verify with mocks.
+ */
+#ifndef HEADER_TLS_INTERNAL_MOCK_H
+#define HEADER_TLS_INTERNAL_MOCK_H
+
+#include "tls_verify.h"
+
+/* Empty macro definitions to mock out the tls_set_error* functions
+ */
+
+#define tls_set_errorx(ctx, errmsg, name) {}
+#define tls_set_error(ctx, errmsg, name) {}
```

Comment:
Strictly speaking, these are not empty macros. Rather they expand to `{}`. I would convert them as empty macro as part of merging the pull request.